app/helpers/solidus_viabill/api/checkout_helper.rb

```diff
@@ -6,7 +6,7 @@ module CheckoutHelper
       VIABILL_PROTOCOL = '3.1'
       VIABILL_STATUS = %w[CANCELLED APPROVED REJECTED].freeze
       def build_checkout_request_body(order, payment_method_id, frontend)
-        gateway = SolidusViabill::Gateway.new({})
+        gateway = SolidusViabill::Gateway.new
```

Comment:
This commit can be fixed up with the previous one, right?

Reply:
removing `({})` yes, but the other changes are there from things that were present much earlier - should this commit be squashed even in this case?